dot-net-sdk/dto/AssignmentLogData.cs

```diff
@@ -1,8 +1,9 @@
 using System.Reflection.Metadata.Ecma335;
+using System.Runtime.Serialization;
 
 namespace eppo_sdk.dto;
 
-public record AssignmentLogData
+public record AssignmentLogData : ISerializable
```

Comment:
ISerializable lets the object be passed directly to Segment analytics SDK, and possibly others.

Reply:
Great thinking!